meinberlin/apps/cms/management/commands/update_storefront_counts.py

```diff
@@ -0,0 +1,58 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from adhocracy4.comments.models import Comment
+from adhocracy4.modules.models import Item
+from adhocracy4.projects.enums import Access
+from adhocracy4.projects.models import Project
+from meinberlin.apps.cms.models import storefronts
+from meinberlin.apps.plans.models import Plan
+
+
+class Command(BaseCommand):
+    help = "Update item and project counts for the storefront."
+
+    def handle(self, *args, **options):
+        for storefront in storefronts.Storefront.objects.all():
+            storefront.num_entries = self.get_num_entries_count()
+            storefront.num_projects = self.get_num_projects_count()
+            storefront.save()
+        for item in storefronts.StorefrontItem.objects.filter(district__isnull=False):
+            district = item.district
+            item.district_project_count = self.get_district_project_count(district)
+            item.save()
+
+    def get_num_entries_count(self):
```

Comment:
Also here, couldn't we use the methods from models/storefronts.py here? So either import them here or move them to the Storefront class?